src/pushInfo/index.tsx

```diff
@@ -9,26 +9,36 @@ interface PushInfoProps {
   readonly icon?: React.ReactNode
   readonly headline: string
   readonly content?: string
+  readonly onAnimationEnd?: Function
 }
 
-const PushInfo = ({ className, icon, headline, content }: PushInfoProps) => (
-  <div className={cc(['kirk-pushInfo', className])}>
-    {icon && (
-      <div className="kirk-pushInfo-icon">{icon}</div>
-    )
+class PushInfo extends Component<PushInfoProps> {
+  componentDidMount() {
+    if (this.props.onAnimationEnd) {
+      setTimeout(this.props.onAnimationEnd, 1000)
```

Comment:
Ok, too bad we can't really have the animation time as a prop, given it's split into 2 steps in the css (700ms + 300ms)